src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -157,6 +157,23 @@ public static String getCountryCode() {
     return getProp("countryCode");
   }
 
+  /**
+   * @return null if window is supposed to be maximised, Dimension if any other size is demanded
+   */
+  public static Dimension getBrowserSize() {
+    String size = getProp("browserSize");
+    if (StringUtils.isNotBlank(size) || "max".equals(size) || size.split("x").length == 2) {
+      if ("max".equals(size)) {
+        return null;
```

Comment:
`null` doesn't seem to be proper value for maximised window, can you return calculated real dimensions? or at least information that it is maximised window?